### ISSUE TITLE
App Couldn't connect to it's vpn

### ISSUE THREAD

**Issue by shaibow** [haxpor/Potatso#16]
I've pulled last version of app and went through all steps successfully !
But when i build app on my device and add server to it , when i press connect button it rounds and nothing happens and it remains connect again !
I've bought the original app from Appstore and it works with my server!
What should i do after creating App IDs and creating provisions and adding to project !?
** the github app successfully create it's own VPN on iPhone but can't connect to it .

**Comment by haxpor** [comment]
Firstly, please let me know your ios version ?

If it's ios `9.x`, this probably relate to #14 . We now know it's a known issue.

**Comment by shaibow** [comment]
My ios Version is 10.2

**Comment by haxpor** [comment]
Did you happen change bundle identifier?

**Comment by shaibow** [comment]
Yes, Exactly as the readme file said.
<img width="1141" alt="screen shot 2017-03-19 at 7 33 07 pm" src="https://cloud.githubusercontent.com/assets/12056542/24082446/06cc9544-0cdb-11e7-926b-a394627a1fa5.png">
<img width="1135" alt="screen shot 2017-03-19 at 7 32 55 pm" src="https://cloud.githubusercontent.com/assets/12056542/24082447/06cd9372-0cdb-11e7-9f7d-f084c67a2870.png">
<img width="1138" alt="screen shot 2017-03-19 at 7 32 25 pm" src="https://cloud.githubusercontent.com/assets/12056542/24082448/06d0806e-0cdb-11e7-9071-eaf8f674cf00.png">
<img width="1141" alt="screen shot 2017-03-19 at 7 32 15 pm" src="https://cloud.githubusercontent.com/assets/12056542/24082450/06d19706-0cdb-11e7-88de-384107350c3b.png">
<img width="1136" alt="screen shot 2017-03-19 at 7 32 03 pm" src="https://cloud.githubusercontent.com/assets/12056542/24082449/06d09d2e-0cdb-11e7-9961-a61d9d021cc3.png">

And as You can see in pictures everything is set correctly and no errors!

**Comment by haxpor** [comment]
Please try to change content in `Potatso.m` file as seen in https://github.com/haxpor/Potatso/issues/13#issuecomment-284955274 to return value with your shared group identifier in this case as seen in your screenshots is `group.com.iavashaki.potatsovpn`.

Let me know whether it works or not.

**Comment by shaibow** [comment]
Thanks <3
That Worked Like a cake 🍰 
But why ? :D the code was doing the same !

**Comment by haxpor** [comment]
As per my knowledge working with the code, It somewhat needs to operate on the same shared group identifier for those stuff including networking thing.

And I think you refer to old version of README, latest version, there's no need to change (but just in case someone wants to, then this issue might pop up).

I believe if you didn't change all those domains & identifier, it would be working out just fine. But nonetheless, flexibility to set should be there too. First before this, I made it to be customized from bundle identifier without users to look at it, but at last there're an issue (including me too) with the same problem you experienced. Not sure why even checked with the identifier and it's the same, correct value.

I'll look into it more, and decide again whether or not to make this thing more flexible.

Thanks for reporting 👍 

**TLDR;**
For users that changed bundle identifier, iCloud, and App group identifier, thus you need to look at `Potato.m` file and change return value to match your `App Groups` as set in Capabilities tap on Xcode too. You can see screenshots above.
